i18n/fr.json

```
@@ -43,13 +43,23 @@
     "productions": "Productions",
     "logout": "Déconnecter"
   },
-  "actions": {
-    "create": "toevoegen"
-  },
   "pagination": {
     "previous": "Dernier",
     "next": "Prochain"
   },
+  "productions": {
+    "create": "ajouter",
+    "delete": "suprimer",
```

```suggestion
    "delete": "supprimer",
```
You can use google translate for the translations ;)